extContractsGroup.json

```diff
@@ -0,0 +1,41 @@
+{
+        "name": "contivModel",
+                "objects": [
+                {
+                        "name": "extContractsGroup",
+                        "type": "object",
+                        "key": [ "tenantName", "contractsGroupName" ],
+                        "properties": {
+                                "tenantName": {
+                                        "type": "string",
+                                        "title": "Tenant name",
+                                        "description": "Tenant name",
```

Comment:
Can you please add validators(format field) to this property?
Please see endpointGroup.json for how to define `format` regex expressions